diff -aurN nautilus-sendto-0.6.orig/configure.in nautilus-sendto-0.6/configure.in
--- nautilus-sendto-0.6.orig/configure.in	2006-06-24 17:25:13.000000000 +0200
+++ nautilus-sendto-0.6/configure.in	2006-06-24 23:34:16.781417672 +0200
@@ -149,14 +149,8 @@
         [
         AC_SUBST(DBUS_CFLAGS)
         AC_SUBST(DBUS_LIBS)
-	if test -f "$GAJIM_SHARE_DIR"/data/pixmaps/gajim.png ;
-	then
-		enable_gajim=yes
-		AC_DEFINE_UNQUOTED(GAJIM_SHARE_DIR, "$GAJIM_SHARE_DIR", [path to gajim share dir])
-	else
-		echo "Building without Gajim support"
-		enable_gajim=no
-	fi
+	enable_gajim=yes
+	AC_DEFINE_UNQUOTED(GAJIM_SHARE_DIR, "$GAJIM_SHARE_DIR", [path to gajim share dir])
         ],
         [
         echo "Building without Gajim support"
