diff -urN nautilus-sendto-1.1.3/configure.in nautilus-sendto-1.1.3.new/configure.in
--- nautilus-sendto-1.1.3/configure.in	2009-04-03 13:46:28.000000000 +0200
+++ nautilus-sendto-1.1.3.new/configure.in	2009-04-11 13:17:04.159223123 +0200
@@ -195,12 +195,8 @@
 					  [
 					   AC_SUBST(DBUS_CFLAGS)
 					   AC_SUBST(DBUS_LIBS)
-					   if test -f "$GAJIM_SHARE_DIR"/data/pixmaps/gajim.png ; then
-						   enable_gajim=yes
-						   AC_DEFINE_UNQUOTED(GAJIM_SHARE_DIR, "$GAJIM_SHARE_DIR", [path to gajim share dir])
-					   else
-						   enable_gajim=no
-					   fi
+					   enable_gajim=yes
+					   AC_DEFINE_UNQUOTED(GAJIM_SHARE_DIR, "$GAJIM_SHARE_DIR", [path to gajim share dir])
 					   ],
 					   [
 					    enable_gajim=no
